chpl/chpl-service/src/main/java/gov/healthit/chpl/web/controller/ChartDataController.java

```diff
@@ -0,0 +1,60 @@
+package gov.healthit.chpl.web.controller;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import gov.healthit.chpl.domain.Announcement;
+import gov.healthit.chpl.domain.ChartData;
+import gov.healthit.chpl.domain.ChartDataStatType;
+import gov.healthit.chpl.dto.AnnouncementDTO;
+import gov.healthit.chpl.dto.ChartDataDTO;
+import gov.healthit.chpl.dto.ChartDataStatTypeDTO;
+import gov.healthit.chpl.manager.ChartDataManager;
+import gov.healthit.chpl.web.controller.results.AnnouncementResults;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@Api(value="data")
+@RestController
+@RequestMapping(value="/data")
```

Comment:
In the SearchViewController, there are many URLs starting with /data/* and they are all lookup type things (like /data/age_ranges to get the list of available age ranges in the system). I think that this controller might be better with a root @RequestMapping of /statistics; you could move /data/statistics_types into the SearchViewController and use one of the KeyValueModel objects similar to the other calls in there which has fields for name and ID.